README.md

```diff
@@ -5,6 +5,37 @@ Docker Images for RancherOS
 
 Run `make`
 
+## Details
+
+### Multi-arch Dockerfiles
+
+As a pre-cursor to native multi-arch support, we leverage some features of
+[dapper](https://github.com/rancher/dapper).
+
+In the Dockerfiles, you'll see lines like:
+
+```
+FROM rancher/os-centosconsole-base
+# FROM amd64=centos:7 arm64=skip arm=skip
+```
+
+The `rancher/os-centosconsole-base` does not actually exist. Dapper will download the
+arch specific image listed in the commented out `# FROM` line, and tag that, so the
+build can occur.
+
+## Special Entrypoint and CMD
```

Comment:
I really like the fact that we're documenting this, but this README feels like the wrong place for it. This would go really well in a "custom console image" page in our docs.